include/swift/DependencyScan/DependencyScanImpl.h

```diff
@@ -0,0 +1,126 @@
+//===-------------- DependencyScanImpl.h - Swift Compiler -----------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Implementation details of the dependency scanning C API
+//
+//===----------------------------------------------------------------------===//
+#ifndef SWIFT_C_DEPENDENCY_SCAN_IMPL_H
+#define SWIFT_C_DEPENDENCY_SCAN_IMPL_H
+
+#include "swift-c/DependencyScan/DependencyScan.h"
+#include "swift/DependencyScan/DependencyScanningTool.h"
+
+DEPSCAN_BEGIN_DECLS
```

Comment:
Use of these macros here should not be necessary since this header is intended to be consumed by C++ code.

Reply:
Removed, thanks for the catch.